docs/source/kernel_methods_kernels.rst

```diff
@@ -0,0 +1,60 @@
+.. currentmodule:: statsmodels.kernel_methods
+
+Module :py:mod:`.kernels`
+=========================
+
+Kernel base classes
+-------------------
+
+.. autosummary::
+    :toctree: generated/
+
+    kernels.Kernel1D
+    kernels.KernelnD
```

Comment:
It's quite common in geometry though ... you will work on an nD space  with lower case n and upper case D. So unless you feel strongly against this, I would rather keep it.